ads/google/imaVideo.js

```diff
@@ -38,74 +41,45 @@ const PlayerStates = {
-     <path d="M0 0h24v24H0z" fill="none"></path>`,
-  'volume_max': `<path d="M3 9v6h4l5 5V4L7 9H3zm13.5 3c0-1.77-1.02-3.29-2.5-4.03v8.05c1.48-.73 2.5-2.25 2.5-4.02zM14 3.23v2.06c2.89.86 5 3.54 5 6.71s-2.11 5.85-5 6.71v2.06c4.01-.91 7-4.49 7-8.77s-2.99-7.86-7-8.77z"></path>
-     <path d="M0 0h24v24H0z" fill="none"></path>`,
-  'seek': `<circle cx="12" cy="12" r="12" />`,
+  play: (svg) => svg`
+    <svg viewBox="0 0 24 24">
+      <path d="M8 5v14l11-7z"></path>
+    </svg>
+  `,
+  pause: (svg) => svg`
+    <svg viewBox="0 0 24 24">
+      <path d="M6 19h4V5H6v14zm8-14v14h4V5h-4z"></path>
+      <path d="M0 0h24v24H0z" fill="none"></path>
+    </svg>
+  `,
+  fullscreen: (svg) => svg`
+    <svg viewBox="0 0 24 24">
+      <path d="M0 0h24v24H0z" fill="none"></path>
+      <path
+        d="M7 14H5v5h5v-2H7v-3zm-2-4h2V7h3V5H5v5zm12 7h-3v2h5v-5h-2v3zM14 5v2h3v3h2V5h-5z"
+      ></path>
+    </svg>
+  `,
+  muted: (svg) => svg`
+    <svg viewBox="0 0 24 24">
+      <path
+        d="M16.5 12c0-1.77-1.02-3.29-2.5-4.03v2.21l2.45 2.45c.03-.2.05-.41.05-.63zm2.5 0c0 .94-.2 1.82-.54 2.64l1.51 1.51C20.63 14.91 21 13.5 21 12c0-4.28-2.99-7.86-7-8.77v2.06c2.89.86 5 3.54 5 6.71zM4.27 3L3 4.27 7.73 9H3v6h4l5 5v-6.73l4.25 4.25c-.67.52-1.42.93-2.25 1.18v2.06c1.38-.31 2.63-.95 3.69-1.81L19.73 21 21 19.73l-9-9L4.27 3zM12 4L9.91 6.09 12 8.18V4z"
+      ></path>
+      <path d="M0 0h24v24H0z" fill="none"></path>
+    </svg>
+  `,
+  volumeMax: (svg) => svg`
+    <svg viewBox="0 0 24 24">
+      <path
+        d="M3 9v6h4l5 5V4L7 9H3zm13.5 3c0-1.77-1.02-3.29-2.5-4.03v8.05c1.48-.73 2.5-2.25 2.5-4.02zM14 3.23v2.06c2.89.86 5 3.54 5 6.71s-2.11 5.85-5 6.71v2.06c4.01-.91 7-4.49 7-8.77s-2.99-7.86-7-8.77z"
+      ></path>
+      <path d="M0 0h24v24H0z" fill="none"></path>
+    </svg>
+  `,
 };
 
-const bigPlayDivDisplayStyle = 'table-cell';
-
-// Div wrapping our entire DOM.
-let wrapperDiv;
-
-// Div containing big play button. Rendered before player starts.
-let bigPlayDiv;
-
-// Div contianing play button. Double-nested for alignment.
-let playButtonDiv;
-
-// Div containing player controls.
-let controlsDiv;
-
-// Wrapper for ad countdown element.
-let countdownWrapperDiv;
-
-// Div containing ad countdown timer.
-let countdownDiv;
-
-// Div containing play or pause button.
-let playPauseDiv;
-
-// Div containing player time.
-let timeDiv;
-
-// Node containing the player time text.
-let timeNode;
-
-// Wrapper for progress bar DOM elements.
-let progressBarWrapperDiv;
-
-// Line for progress bar.
-let progressLine;
-
-// Line for total time in progress bar.
-let totalTimeLine;
-
-// Div containing the marker for the progress.
-let progressMarkerDiv;
-
-// Div for fullscreen icon.
-let fullscreenDiv;
-
-// Div for mute/unmute icon.
-let muteUnmuteDiv;
-
-// Div for ad container.
-let adContainerDiv;
-
-// Div for content player.
-let contentDiv;
-
-// Content player.
-let videoPlayer;
+// References to rendered elements. See renderElements().
+let elements;
```

Comment:
We collect `elements` as a result of `htmlRefs`, so we remove their corresponding globals. 

In turn, some removed globals do not have a corresponding `ref` in the static template since we don't need to reference them from Javascript anymore.